### DISCUSSION TITLE
Synchronisation with BAM is completely broken

### DISCUSSION THREAD

**Discussion by IRainman** [BOINC/boinc#5822] [General]
**Describe the bug**
I use BAM to manage my hosts, and after upgrading from boinc_7.24.1_windows_x86_64.exe to boinc_8.0.4_windows_x86_64.exe clients entirely ruins settings from BAM and don't work properly (((

**Steps To Reproduce**
1. Connect BAM account to the client.
2. Upgrade from Boinc 7 to Boinc 8.
3. Get many problems with wrong flags for projects, wrong list of the project etc.

**Expected behavior**
After upgrading, all behaviour needs to be same.

**System Information**
 - OS: Windows 10 and 2008R2, both 64 bit
 - BOINC Version: 8.0.4 64 bit

**Comment by AenBleidd** [comment]
This is a known issue.
Unfortunately, according to the observations, the issue on the side of BAM! because from the BOINC side there was no changes that could lead to such issues.
I'm moving this to the Discussion since at the moment there is nothing we can do from our side to resolve this issue.
